src/pages/api/fetch-datasets.ts

```diff
@@ -0,0 +1,14 @@
+import type {NextApiRequest, NextApiResponse} from 'next';
+import {DatasetFetcher} from '@/lib/dataset-fetcher';
+
+const datasetFetcher = new DatasetFetcher({
+  endpointUrl: process.env.SEARCH_PLATFORM_ELASTIC_ENDPOINT_URL as string,
+});
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const datasets = await datasetFetcher.search();
```

Comment:
The name `datasets` suggests that `search()` returns datasets, but it actually returns a search result, of which the datasets are a part. Change the name?